### ISSUE TITLE
require-hyphen-before-param-description: False-positive for hyphenated first word

### ISSUE THREAD

**Issue by kevinoid** [gajus/eslint-plugin-jsdoc#665]
## Expected behavior

`jsdoc/require-hyphen-before-param-description: ["error", "never"]` is not triggered by `@param {!Array<string>} argv Command-line arguments.`.

## Actual behavior

    index.js
      3:0  error  There must be no hyphen before @param description  jsdoc/require-hyphen-before-param-description
    
    ✖ 1 problem (1 error, 0 warnings)
      1 error and 0 warnings potentially fixable with the `--fix` option.

## ESLint Config

```json
{
  "plugins": [
    "jsdoc"
  ],
  "rules": {
    "jsdoc/require-hyphen-before-param-description": ["error", "never"]
  }
}
```

## ESLint sample

```js
/** Entry point for module.
 *
 * @param {!Array<string>} argv Command-line arguments.
 */
function main(argv) {
};
```

## Environment

- Node version: v12.19.0
- ESLint version v7.17.0
- `eslint-plugin-jsdoc` version: 30.7.11

## Additional Notes

I am also able to reproduce the issue in `30.7.10`, but not `30.7.9`.  It appears to be a regression in `30.7.10`.

Thanks,
Kevin

**Comment by kevinoid** [comment]
Thanks for the lightning-fast fix @brettz9!

**Comment by brettz9** [comment]
Thanks for the lightning fast report! :-)
